packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -0,0 +1,693 @@
+            @click="removeItem({ product })"
+          >
+            Clear wishlist
+          </button>
+        </li>
+      </ul>
+      <span>
+        {{ totals.total }}
+      </span>
+      <span>
+        {{ totalItems }}
+      </span>        
+    </div>
+  // ...
+</template>   
+<script>
+  import { computed } from '@vue/composition-api';
+  import { useWishlist, wishlistGetters } from '{INTEGRATION}';
+  import { onSSR } from '@vue-storefront/core';
+  export default {
+    setup() {
+      const {
+        wishlist, 
+        removeItem, 
+        loading
+      } = useWishlist();
+    
+    const products = computed(() => wishlistGetters.getItems(wishlist.value));
+    const totals = computed(() => wishlistGetters.getTotals(wishlist.value));
+    const totalItems = computed(() => wishlistGetters.getTotalItems(wishlist.value));
+
+      return {
+        products,
+        totals,
+        totalItems,
+        removeItem,
+        loading
+      };
+    }
+  };
+</script>
+```
+
+## Checking if an item is on the wishlist 
+
+To check if a product is already on the wishlist pass it to `isInWishlist` method:
+
+```vue
+<template>
+  // ...
+    <ul>
+      <li
+        v-for="(product, i) in products" :key="i"
+      >
+        <div
+          :isAddedToWishlist="isInWishlist({ product })"
+        >
+        </div>
+      </li>
+    </ul>
+  // ...
+</template>    
+<script>
+  import { computed } from '@vue/composition-api';
+  import { useWishlist} from '{INTEGRATION}';
+  export default {
+    props: {
+      products: {
+        type: Array,
+        required: true
+      }
+    },
+    setup() {
+      const {
+        cart,
+        isInWishlist, 
+      } = useWishlist();
+
+      return {
+        isInWishlist, 
+      };
+    }
+  };
+</script>
+```
+
+## Removing all wishlist items at once
+
+Cleaning the wishlist can be achieved by `clear` property.
+
+```vue
+<template>
+  // ...
+    <div>
+      <ul>
+        <li
+          v-for="(product, i) in products" :key="i"
+        >
+          // ...
+        </li>
+      <ul>
+      <button
+        @click="clear()"
+      >
+        Clear wishlist
+      </button>
+    </div>
+  // ...
+</template>   
+<script>      
+  import { computed } from '@vue/composition-api';
+  import { useWishlist, wishlistGetters } from '{INTEGRATION}';
+
+  export default {
+    setup() {
+      const {
+        wishlist,
+        clear, 
+      } = useCart();
+      const products = computed(() => wishlistGetters.getItems(wishlist.value));
+
+      return {
+        wishlist,
+        products,
+        clear 
+      };
+    }
+  };
+</script>
+```
+
+## Summary
+
+### Common usage example
+
+In the following examples, you can analyze how both composables are used in the simple use case. There are all above mentioned basic scenarios used in three main components: a product list, a cart, and a wishlist. It can be your basis for building real-life application implementation.    
```

Comment:
I believe we could simplify wording here a bit. Also, it's not completely clear from the context which scenarios we mean.

```suggestion
Here is an example of how both composables can be used in real-life applications. We have three components: a product list, a cart, and a wishlist. In the examples, we cover the logic of adding/removing items from the wish list and cart, working with getters, and loading state
```

I might be missing something in the description 😅